src/mrack/host.py

```diff
@@ -116,25 +115,24 @@ def provider(self):
         return self._provider
 
     @property
-    def id(self):
+    def host_id(self):
         """Get provider host id."""
-        return self._id
+        return self._host_id
 
     @property
     def name(self):
         """Get host name."""
         return self._name
 
     @property
-    def ips(self):
+    def ip_addrs(self):
```

Comment:
Just a rant: This rule, I'm assume minimal number of chars in variable/attribute name, is IMHO stupid - e.g. in cases like this.